README.md

```diff
@@ -20,47 +12,61 @@ Pick a theme that the team likes. A good place for inspiration is the [Wordpress
 Each team member is going to be responsible for the content of a different page, so decide on the name and content for each of those pages. Some possible options are...
 
 1. Home page
-1. Employee listing
 1. Company history
 1. Company culture/mission
 1. Job listings
 1. Company awards
 
-Regardless of what pages each team member works on, each team must have a Products page for their site that the team will collaborate on. See more details about the product page below.
+Regardless of what pages each team member works on, each team must have a Products page and an Employee page for their site that the team will collaborate on. See more details about the product page below.
 
 ### Git
 Your team is going to learn how to use git to enable good teamwork when building a software product.
 
-1. Now that everyone has joined the same team, everyone clone the Github Classroom project into your current directory - which should be `group-projects` if you ran the setup commands.
+1. Now that everyone has joined the same team, everyone clone the Github Classroom project into your current directory - which should be `groupprojects` if you ran the setup commands.
 1. `cd` into the directory that got created.
 1. Before anyone on the team writes a single line of code, the team will decide on a branch naming strategy. Branches should be descriptive of the work that is going to be done on that branch. Some teams also decide to add the developer's information to the branch name, such as initials. For example, a developer named Jasmine Sheboygen is going to work on the job listings page for the web site. A good name for the branch would be `js-job-listing-page`.
 1. Again, before you write any code, one team member volunteer to create the  ignore file on a branch.
     1. `touch .gitignore`
-    1. `git add .`
+    1. `git add .gitignore`
     1. `git commit -m "Initial commit with .gitignore file"`
     1. `git push origin YOUR BRANCH NAME`
-1. Now make a PR and get your team to approve it and merge into master.
-1. Now everyone should `git pull origin master`.
+1. Now make a PR and get your team to approve it :+1: and merge into master on GitHub.
+1. Now everyone should `git pull origin master` (from their local master branch).
 1. That's the last commit you make on the master branch. At this point, each teammate will create their branch.
 
-Now everyone will work on their page on those individual branches. Only when teammates feel that the work is complete, the teammate will push up their branch to Github and create a pull request. One of the other teammates must review the pull request and give a thumbs up (Just type :+1 in the comment box).
+Now everyone will work on their page on those individual branches. Once a teammates feels that their work is complete, the teammate will push up their branch to Github and create a pull request. One of the other teammates must review the pull request and give a thumbs up (Just type :+1 in the comment box).
```

Comment:
maybe elaborate a little more on reviewing PRs - look over the code on the PR.  pull the code down and run it to make sure it works.